static/docs/tutorials/versioning.md

```diff
@@ -1,52 +1,47 @@
 # Tutorial: Versioning
 
-> Reading time is 10-13 minutes. Running the training is 30-40 minutes
-> (including downloading the dataset). Running the code is optional, and reading
-> alone should be enough to learn how to manipulate different versions of data
-> and models.
+> Reading time: 10-15 min.<br> Downloading dataset and running training: 30-40
+> min.
```

Comment:
Question for @shcheklein: Is this block useful? Should we add it to the other tutorials? Notice how I reduced it much, to:

> Reading time: 10-15 min.
> Downloading dataset and running training: 30-40 min.

BTW, how do we know this is a 10-15 min. read? I think that if you're trying to follow it, running the commands, etc. it's going to be a much longer read.

p.s. @dashohoxha you may be interested in this PR in general. I think you're the previous person to make relevant edits this file.

Reply:
> p.s. @dashohoxha you may be interested in this PR in general. I think you're the previous person to make relevant edits this file.

@jorgeorpinel I have [never touched this file](https://github.com/iterative/dvc.org/blame/b179af09c15897149ca3f768b51f3725dcb4bc33/static/docs/tutorials/versioning.md), except for changing its location.
What I have done is that I have built an [interactive tutorial](https://katacoda.com/dvc/courses/tutorials/versioning) based on the scenarios of this one.

In general, I may review a document and try to improve it, but I find it hard to review the reviews of reviews of somebody else, because I loose the context and I don't get the whole picture. Sorry, I have my own limitations. I am also not good at multitasking.

Reply:
Yeah no worries it was just a suggestion in case you were interested.

Reply:
@jorgeorpinel tbh, since it's not consistent across other docs I would remove it altogether.

Reply:
I'll remove it on a future PR.